packages/gatsby-link/README.md

```diff
@@ -38,3 +39,17 @@ render () {
   </div>
 }
 ```
+
+## Programmatic navigation
+
+For cases when you can only use event handlers for navigation, you can use `navigateTo`:
+
+```jsx
+import { navigateTo } from "gatsby-link"
+
+render () {
+  <div onClick={navigateTo('/example')}>
```

Comment:
Yeah, it had to be something like [this](https://codepen.io/rah98/pen/MoPXjy).
I'll submit a small PR for that.